tests/test_instances.py

```diff
@@ -46,13 +46,14 @@ def test_fields(self, instance, i_data, cls, mod):
         assert -90 <= instance.meta['latitude'] <= 90
         assert -180 <= instance.meta['longitude'] <= 180
 
-        assert 'company' in instance.meta
-
-        company = instance.meta['company']
-        err = "[company] expected list, found '%s' for %s" % (
-            type(company).__name__, company
-        )
-        assert isinstance(instance.meta['company'], list), err
+        if instance.meta['company'] is not None:
+            assert 'company' in instance.meta
+
+            company = instance.meta['company']
+            err = "[company] expected list, found '%s' for %s" % (
+                type(company).__name__, company
+            )
+            assert isinstance(instance.meta['company'], list), err
```

Comment:
let me know if this is weird. tests were failing before

Reply:
Tests fail here when an instance does not have a company, which should always have.